var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -34,270 +52,289 @@ class Vasp(MakefilePackage):
         when="+vaspsol",
     )
 
-    variant("openmp", default=False, description="Enable openmp build")
-    with when("+openmp"):
-        conflicts("^fftw~openmp")
-        conflicts("^amdfftw~openmp")
-        conflicts("^amdblis threads=none")
-        conflicts("^amdblis threads=pthreads")
-        conflicts("^openblas threads=none")
-        conflicts("^openblas threads=pthreads")
+    variant("openmp", default=False, when="@6:", description="Enable openmp build")
 
-    variant("scalapack", default=False, description="Enables build with SCALAPACK")
+    variant("scalapack", default=False, when="@:5", description="Enables build with SCALAPACK")
 
     variant("cuda", default=False, description="Enables running on Nvidia GPUs")
-    variant("fftlib", default=False, description="Enables fftlib build")
-    with when("+fftlib"):
-        conflicts("@:6.1.1", msg="fftlib support started from 6.2.0")
-        conflicts("~openmp", msg="fftlib is intended to be used with openmp")
+    variant("fftlib", default=True, when="@6: +openmp", description="Enables fftlib build")
```

Comment:
The 'when' condition here is not consistent to the old logic. Should it be `when="@6.2.0: +openmp"?

Reply:
Wiki and source suggest fftlib is available in 6.1, but there needs to be some additional glue to build it in the version of 6.1 that I have, seemingly. There's enough special cases in this as is, so I'll bump that to 6.2.